### ISSUE TITLE
Unable to download video subtitles for 'live_chat': Expecting value: line 1

### ISSUE THREAD

**Issue by Yannf** [toolforge/video2commons#174]
Getting this error message even though subtitles are unchecked: "An exception occurred: DownloadError: b"ERROR: Unable to download video subtitles for 'live_chat': Expecting value: line 1 column 1 (char 0)"
[https://commons.wikimedia.org/w/index.php?title=Commons_talk:Video2commons&oldid=826276202#Subtitle_error%2C_even_when_subtitles_are_unchecked].
I get this error repeatedly.

**Comment by royjor2016** [comment]
(Battleship "Potemkin") yt-dlp shows
```
[youtube] Extracting URL: https://www.youtube.com/watch?v=a_bkBbrdyyw
[youtube] a_bkBbrdyyw: Downloading webpage
[youtube] a_bkBbrdyyw: Downloading ios player API JSON
[youtube] a_bkBbrdyyw: Downloading android player API JSON
[youtube] a_bkBbrdyyw: Downloading m3u8 information
a_bkBbrdyyw has no automatic captions
[info] Available subtitles for a_bkBbrdyyw:
Language  Formats
live_chat json
```

we need a way to ignore this live_chat thing.

**Comment by royjor2016** [comment]
potential positions for rewrite
https://github.com/toolforge/video2commons/blob/0a7eb3ecdc84bf6014cbc52622e4a3be1fd583b5/video2commons/backend/download/__init__.py#L58

if i understand correctly, this portion of code passes these params to https://github.com/yt-dlp/yt-dlp/blob/master/yt_dlp/YoutubeDL.py .
then, they deal with `allsubtitles` like this
https://github.com/yt-dlp/yt-dlp/blob/6a9c7a2b52655bacfa7ab2da24fd0d14a6fff495/yt_dlp/YoutubeDL.py#L524
https://github.com/yt-dlp/yt-dlp/blob/6a9c7a2b52655bacfa7ab2da24fd0d14a6fff495/yt_dlp/YoutubeDL.py#L2970

i think, asking on stack overflow or reddit, "how to download all subs except live_chat while using ytdlp", will help.
or, i probably just found a method.

**Comment by chicocvenancio** [comment]
Fixed in #175 by @royjor2016
